_src/pattern-library/sass/components/_forms.scss

```diff
@@ -0,0 +1,424 @@
+// ------------------------------
+// edX Pattern Library: Components - Forms
+
+// About: Contains base styling for forms
+
+// #GLOBAL
+// #INPUT TEXT
+// #INPUT RADIO/CHECKBOX
+// #SELECT
+// #TEXTAREA
+// #SLIDERS
+// #PROGRESS
+
+// ----------------------------
+// #GLOBAL
+// ----------------------------
+.fieldset {
+    @include clearfix;
+    margin-bottom: spacing-vertical(mid-small);
+
+    .group-legend {
+        margin-bottom: spacing-vertical(x-small);
+    }
+
+    .field-wrapper {
+        margin-bottom: spacing-vertical(base);
+
+        .helper-text {
+            font-size: font-size(small);
+            color: $helper-text;
+        }
+    }
+
+    &.form-group {
+
+        .field-wrapper { // Generally used for radios and checkboxes
+            margin-bottom: spacing-vertical(x-small);
```

Comment:
Hmm, I'm not sure I agree. Here's what happens with `small`:

![screen shot 2015-06-09 at 12 53 20 pm](https://cloud.githubusercontent.com/assets/2112024/8064146/df424660-0ea6-11e5-9368-6e753499d961.png)

![screen shot 2015-06-09 at 12 53 23 pm](https://cloud.githubusercontent.com/assets/2112024/8064147/e1e1cfee-0ea6-11e5-93e6-ff06628690f5.png)

Kinda looks too much, don't you think?